autoextract_spiders/spiders/autoextract_article.py

```diff
@@ -35,7 +46,12 @@ def parse_source(self, response: HtmlResponse):
             self.crawler.stats.inc_value('sources/rss')
             meta = {'source_url': response.meta['source_url'], 'feed_url': feed_url}
             self.crawler.stats.inc_value('x_request/feeds')
-            yield Request(feed_url, meta=meta, callback=self.parse_feed, errback=self.errback_feed)
+            yield Request(
+                feed_url,
+                meta=meta,
+                callback=self.parse_feed,
+                errback=self.errback_feed,
+                dont_filter=True)
```

Comment:
I believe we shouldn't filter source/feed level requests to have incremental crawls. Or we could have it switchable via spider argument.

Reply:
Yes I agree at this point we shouldn't filter the request, because we want to parse the feed everytime.